modules/admin_manual/pages/issues/impersonate_users.adoc

```diff
@@ -31,37 +32,32 @@ see.
 [[ending-an-impersonation]]
 == Ending an Impersonation
 
-When you’re ready to stop impersonating the user, log out and you will
+When you're ready to stop impersonating the user, log out and you will
 return to your normal user session.
 
 [[restrict-impersonation-to-groups-group-administrators]]
 == Restrict Impersonation to Groups & Group administrators
 
-As a security measure, the application lets ownCloud administrators
-restrict the ability to impersonate users to administrators of specific
-groups. When enabled and configured, only a group’s administrator can
-impersonate members of their group.
+As a security measure, the application lets ownCloud administrators restrict the ability to impersonate users to:
 
-For example, if an ownCloud administrator restricts user impersonation
-only to the group: `group1', then *only* `group1'’s administrators can
-impersonate users belonging to `group1'. No other users can impersonate
-other users.
+- All group administrators. 
+- Specific group administrators. 
+
+When enabled and configured, only a group's administrator can impersonate members of their group.
+For example, if an ownCloud administrator restricts user impersonation only to the group: `group1`, then *only* `group1`'s administrators can impersonate users belonging to `group1`. 
+No other users can impersonate other users.
 
 NOTE: ownCloud administrators can always impersonate all users of an ownCloud instance 
 when the application is installed.
 
-To enable this option, in the administrator settings panel
-(`administrator -> Settings -> Admin`) in the "**User Authentication**"
-section, you’ll see a section titled: "**Impersonate Settings**"; which you can see below.
+By default, when the Impersonate app is installed, *all* group admins will be allowed to impersonate users within the groups they administer.
```

Comment:
By default, when impersonate app is installed, only admin user ( or say administrator) can only impersonate any users. Only if any of the radio buttons are clicked 
![impersonate_settings](https://user-images.githubusercontent.com/3600427/53474018-ffee7f80-3a91-11e9-8c29-381a4f11fce2.png)
the behaviour changes according to the selection.

Reply:
Thanks for the clarification.

Reply:
When I installed the latest version, "_Allow group admins of specific groups to impersonate the users within those groups_" was already selected. Secondly, what do you mean by "_( or say administrator)_"?